examples/tech_meetings/20240403_run_without_ini_files_DIY.ipynb

```diff
@@ -271,6 +271,54 @@
     "_runner = AnalysisRunnerFactory.get_runner(_config_wrapper)\n",
     "_runner.run(_config_wrapper.analysis_config)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.3. Create Ra2ce Handler without `ini` files - after #389\n",
+    "\n",
+    "In this example the network and analysis configuration are read from file before creation of the `Ra2ceHandler`.\n",
+    "This can be replaced by any other way of creating your configuration, e.g. by creating it from scratch by `NetworkConfigWrapper()` and `AnalysisConfigWrapper()`.\n",
```

Comment:
Do you mean?
```suggestion
    "This can be replaced by any other way of creating your configuration, e.g. by creating it from scratch by `NetworkConfigData()` and `AnalysisConfigData()`.\n",
```

Reply:
Correcte